src/games/ColorGame.js

```diff
@@ -0,0 +1,499 @@
+import React, { useState, useEffect } from 'react';
+import "../css/ColorGame.css";
+import { recordData } from "./GameDataRecorder";
+import { GAMES_ENUM } from "../constants/GamesConstants";
+
+
+
+export default function ColorGame({ advanceStateFunction }) {
+    const colors_value = [
+        '#0000FF', '#FF0000', "#00FF00", "#FFFF00", "#FFA500", "#A020F0"
+        //Blue, Red, Green, Yellow, Orange, Purple
+    ]
+    const colors_names = [
+        //'#0000FF', '#FF0000', "#00FF00", "#FFFF00", "#FFA500", "#A020F0"
+        "Blue", "Red", "Green", "Yellow", "Orange", "Purple" 
+    ]
+
+	const GameState = Object.freeze({
+		Initial: Symbol("Initial"), 
+		InProgress: Symbol("In Progress"),
+		GameOver: Symbol("Game Over")
+	});
+
+	const [startTime, setStartTime] = useState(0);
+    const [endTime, setEndTime] = useState(0);
+    const [timeDiffList, setTimeDiffList] = useState([]);
+
+
+
+	const questions = [
```

Comment:
nitpick, but this should array of objects should either be moved to its own file in the `constants/` directory or *at least* moved outside the scope of the ColorGame component. as-is this entire object is redefined each time the component re-renders. Though performance hasn't been an issue for us so far, this has the potential to hinder performance.

Reply:
I think I fixed both this and the next one in my latest commit. Moved this one outside of the loop and the other one was not being used.